nbconvert/filters/strings.py

```diff
@@ -188,14 +188,59 @@ def get_lines(text, start=None,end=None):
     # Return the right lines.
     return "\n".join(lines[start:end]) #re-join
 
-def ipython2python(code):
+def _mpl_magic_regex_generator(forbidden_backends=None):
+    """Create regex for detecting incompatible backends in mpl magics.
+    
+    Parameters
+    ----------
+
+    forbidden_backens: List
+        List of strings, each of which is a forbidden matplotlib backend. 
+    """
+    if forbidden_backends is None:
+        forbidden_backends = ["notebook", "inline"]
```

Comment:
The code elsewhere is talking about GUI backends, but I would use that term to refer to e.g. the Qt4 and GTK backends, which work with GUI toolkits.